### DISCUSSION TITLE
Colorblind friendly updates

### DISCUSSION THREAD

**Discussion by SpoonOfTruth** [f1multiviewer/issue-tracker#116] [Feature requests] [answered]
I’m colorblind and I’m the live timing screen, the color for Haas and Mercedes looks almost the same to me. Maybe we could get the white square on the drivers last name like RedBull or Ferrari but for Haas so I can tell the difference more easily. 
Also maybe having a stronger contrast option for the segments, they look very similar to my eyes.

**Comment by JJWatMyself** [comment]
Resources that may be helpful.

https://uxdesign.cc/color-blindness-in-user-interfaces-66c27331b858?gi=3c0cee4e1b32
https://www.getfeedback.com/resources/ux/how-to-design-for-color-blindness/
https://www.secretstache.com/blog/designing-for-color-blind-users/

Some pics:
![image](https://user-images.githubusercontent.com/22590445/178152340-f76c45cf-1e14-4213-a71f-ba9ecc4b53df.png)
![image](https://user-images.githubusercontent.com/22590445/178152347-f131bfcb-de65-4dfd-9902-3c4ac23ae4e1.png)
![image](https://user-images.githubusercontent.com/22590445/178152352-7afa6ba3-5721-42ec-9cf6-cb87ae100773.png)

**Comment by f1multiviewer** [comment, marked as answer]
This has shipped, thanks for testing! 🙏
